app/src/main/java/org/mozilla/focus/utils/SupportUtils.kt

```diff
@@ -49,7 +49,7 @@ object SupportUtils {
         WHATS_NEW_KLAR("whats-new-firefox-klar-android"),
         SEARCH_SUGGESTIONS("search-suggestions-focus-android"),
         ALLOWLIST("focus-android-allowlist"),
-        STUDIES("how-opt-out-studies-firefox-android")
+        STUDIES("how-opt-out-studies-firefox-focus-android")
```

Comment:
It seems that this page doesn't exist

<img src="https://user-images.githubusercontent.com/11731652/147048833-ba38060d-0fa5-41cc-bcc9-dd3bca9c9aaa.png" width="30%"/>

Reply:
@kelimuttu Do you know what would be the correct link for this or could you point me to someone else who might know? Thank you!

P.S.: I searched and I could not find a topic on this for Focus Android. One for iOS does exist (https://support.mozilla.org/en-US/kb/studies-focus-ios).

Reply:
Hi @mcarare I checked and confirmed that we don't have one for Focus Android. But I just created [this one](https://support.mozilla.org/en-US/kb/studies-focus-android) for placeholder. Feel free to use that link and I'll tell our content manager to work on the article later. 

Will that work for you?

Reply:
Great! Thank you!

Reply:
Sure thing